### ISSUE TITLE
Generate html with a mock server

### ISSUE THREAD

**Issue by monika-hrda** [concourse/dutyfree#28]
Set up a development server process to help us quickly develop layout and css 
- [x] create mock server for gh readme endpoints
- [x] create `sample-resources.yml`
- [x] `go run main.go <some-folder> sample-resources.yml`
- [x] start http server on `<some-folder>`
- [ ] watch for changes in go, css and template files

**Comment by carlo-colombo** [comment]
at the moment running `dev.go`, 
1. builds dutyfree
1. start a github mock server serving a static readme for all the resources
1. run it against a small `resources.yml` (embedded in `dev.go`) and targeting a temp folder
1. run a static webserver on port 3000 from the temp folder (with dutyfree as context path to replicate github.io serving condition) 


We were planning to build into the dev.go the feature of watching for files change and rebuild but we did not have time/we got stuck. Some experiment can be found in branch `dev-mode`.

During development we used http://eradman.com/entrproject/ to watch for file change and restart `dev.go` 

`ag -l | entr -r go run dev/dev.go`
